app/controllers/project_controller.rb

```diff
@@ -4,4 +4,20 @@ class ProjectController < ApplicationController
   def index
     @projects = Project.all
   end
+
+  def new
+    @project = Project.new
+  end
+
+  def create
+    Project.create(project_params)
```

Comment:
As we are at least validating the prescence, it would be helpful for users to see an error when the validation fails. The GOV.UK Formbuilder has really good support for it: 

https://govuk-form-builder.netlify.app/introduction/error-handling/#input-erb-a-form-with-multiple-errors

Have a look at the RODA controllers to get an idea of the approach: 

https://github.com/UKGovernmentBEIS/beis-report-official-development-assistance/blob/0b75cab3aaa73db9e0c600413eac548279b201b5/app/controllers/staff/organisations_controller.rb#L30